articles-java-jee/src/main/java/com/ibm/cns/articles/control/DataAccessFacade.java

```diff
@@ -0,0 +1,33 @@
+package com.ibm.cns.articles.control;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Produces;
+import javax.inject.Inject;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+@ApplicationScoped
```

Comment:
Hi Adam,
I know it was not introduced by you but is @ApplicationScoped necessary?
BTW: Thx for your refactoring :-)